client/client.go

```diff
@@ -10,29 +11,79 @@ import (
 )
 
 func Start() (*ldap.Conn, error) {
-	conf, err := config.Get()
+	conf := config.Get()
+
+	ldapUrl, err := url.Parse(conf.LdapConfig.Address)
 	if err != nil {
-		slog.With("error", err).Debug("Can't get the config")
 		return nil, err
 	}
 
-	ldapUrl, err := url.Parse(conf.LdapConfig.Address)
-	if ldapUrl.Scheme == "ldap" {
-		slog.With("scheme", ldapUrl.Scheme)
-		if conf.LdapConfig.StartTLS {
-			slog.With("useStartTLS", conf.LdapConfig.StartTLS).Debug("Trying to use startTLS")
-			// We need to use startTLS
-		} else {
-			slog.With("useStartTLS", conf.LdapConfig.StartTLS).Debug("The connection will be INSECURE")
+	tlsConfig := tls.Config{InsecureSkipVerify: conf.LdapConfig.InsecureSkipVerify}
+
+	var c *ldap.Conn = nil
+
+	switch ldapUrl.Scheme {
+	case "ldap":
+		{
+			slog.With("scheme", ldapUrl.Scheme)
```

Comment:
Non stampa niente se poi non fai un `.Debug` o `.Info` o ecc.

Reply:
Unificalo con il debug che hai dentro l'if immagino.
O funziona sta roba per caso?

Reply:
Funziona anche senza `.Debug` o altro. Vuoi comunque che lo unifico? Secondo me ci sta stampare semplicemente lo scheme in alto

Reply:
si unificherei